### PR TITLE
Update skills update

### PR DESCRIPTION
**Pull request by suzkiee**
# 

- [ ] New feature 
- [ ] Refactor
- [x] Bugfix 
- [ ] Documentation

## About

Here's an update to the `updateSkills` mutations so that you can update as many and as little skills as you want at one time. 

## Details

The biggest change I've made to update the attributes so that none of them are required expect for `character_id` and `id` (referring to skills)  

I've added the relevant test and tested it on Graphiql on localhost as well. Can confirm that the mutation runs properly.

### CODE REVIEW

**Comment by andrewvallejo** [comment]
I won't rebase as I don't know enough about Ruby to approve the PR but I super duper appreciate these updates!

File: spec/graphql/mutations/characters/create_character_spec.rb [hunk 1]
```diff
@@ -6,32 +6,32 @@
       @user = User.create(username: "test")
       @query = <<~GQL
                     mutation {
-                      character: createCharacter(
```
Comment (joeray100) [inline spec/graphql/mutations/characters/create_character_spec.rb:9]: Appreciate the consistency fix!

File: app/graphql/mutations/skills/update_skill.rb [hunk 2]
```diff
@@ -1,41 +1,41 @@
 module Mutations
   module Skills
     class UpdateSkill < ::Mutations::BaseMutation
-      argument :astrogation, Integer, required: true
-      argument :athletics, Integer, required: true
-      argument :brawl, Integer, required: true
+      argument :astrogation, Integer, required: false
```
Comment (joeray100) [inline app/graphql/mutations/skills/update_skill.rb:4]: Thanks for adding in this flexibility!

**Review by joeray100 — APPROVED** [review]
Thanks for the work on this!

**Pull request merged**